modules/UI/videolayout/VideoLayout.js

```diff
@@ -857,9 +857,10 @@ var VideoLayout = {
      * @param object
      */
     updateLocalConnectionStats (percent, object) {
-        let resolutions = object.resolution;
+        const { framerates, resolutions } = object;
 
         object.resolution = resolutions[APP.conference.getMyUserId()];
+        object.framerate = framerates[APP.conference.getMyUserId()];
```

Comment:
I just pushed an update that meet your comments.
There is no specific doc for the connection indicator, but this will improve while moving stuff to react.